### DISCUSSION TITLE
Block tooltip info

### DISCUSSION THREAD

**Discussion by BeksOmega** [RaspberryPiFoundation/blockly-keyboard-experimentation#16] [General]
How might we give users access to the information provided in the block's tooltip? Is this redundant and therefor confusing if we also give users access to the help info? (see [related discussion about help info](https://github.com/google/blockly-keyboard-experimentation/discussions/7))

**Comment by rachel-fenichel** [comment]
MDN has documentation on the [ARIA: tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) that we may be able to rely on. I think we should make it possible to force the tooltip open from the context menu if the context menu was opened using a keyboard shortcut or longpress. 

This is also related to https://github.com/google/blockly-keyboard-experimentation/discussions/108, since we would trigger the context menu with Enter.

The tooltip is always plain text and can be displayed inline, while the help option is much more involved: it sometimes opens a new web page or a new pane on the existing page. So I think we need both.
